pkg/combustion/time.go

```diff
@@ -0,0 +1,51 @@
+package combustion
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/suse-edge/edge-image-builder/pkg/fileio"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+	"github.com/suse-edge/edge-image-builder/pkg/log"
+	"github.com/suse-edge/edge-image-builder/pkg/template"
+)
+
+const (
+	timeComponentName = "time"
+	timeScriptName    = "09-time-setup.sh"
```

Comment:
Is there a reason you put this so early in the script execution? I noticed it in `combustion.go` and now here with the prefix.